definitions/divorce/json/CaseTypeTab/CaseTypeTab-deemed-and-dispensed-nonprod.json

```diff
@@ -0,0 +1,84 @@
+[
+  {
+    "LiveFrom": "12/12/2019",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "serviceApplication",
+    "TabLabel": "Service Application",
+    "TabDisplayOrder": 23,
+    "CaseFieldID": "ReceivedServiceApplicationDate",
+    "TabFieldDisplayOrder": 1
+  },
+  {
+    "LiveFrom": "12/12/2019",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "serviceApplication",
+    "TabLabel": "Service Application",
+    "TabDisplayOrder": 23,
+    "CaseFieldID": "ReceivedServiceAddedDate",
+    "TabFieldDisplayOrder": 2
+  },
+  {
+    "LiveFrom": "12/12/2019",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "serviceApplication",
+    "TabLabel": "Service Application",
+    "TabDisplayOrder": 23,
+    "CaseFieldID": "ServiceApplicationType",
+    "TabFieldDisplayOrder": 3
+  },
+  {
+    "LiveFrom": "12/12/2019",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "serviceApplication",
+    "TabLabel": "Service Application",
+    "TabDisplayOrder": 23,
+    "CaseFieldID": "serviceApplicationPaymentType",
+    "TabFieldDisplayOrder": 4
+  },
+  {
+    "LiveFrom": "12/12/2019",
+    "CaseTypeID": "DIVORCE",
+    "Channel": "CaseWorker",
+    "TabID": "serviceApplication",
+    "TabLabel": "Service Application",
+    "TabDisplayOrder": 23,
+    "CaseFieldID": "LabelOutcomeOfServiceApplication",
+    "TabFieldDisplayOrder": 5,
+    "FieldShowCondition" : "ServiceApplicationDecisionDate != \"*\""
```

Comment:
What does this mean?

Reply:
Not to show the **LabelOutcomeOfServiceApplication** label if the value of **ServiceApplicationDecisionDate** is null or empty